### PR TITLE
add: `penguins-eggs-deb`

### PR DESCRIPTION
**Pull request by Anifyuli**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 *.tar.gz
 *.zip
 *.png
+pacstall-programs/*
```
Comment (Elsie19) [inline .gitignore:5]: ..?

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Anifyuli) [reply .gitignore:5]: I want to exclude my venv for installing pre-commit from pip installation, but @oklopfer say it's make some trouble in future. I deleted it but this PR can't update it automatically

Comment (Elsie19) [inline .gitignore:5]:
```suggestion
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/penguins-eggs-deb/penguins-eggs-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name="penguins-eggs-deb"
+arch=("amd64" "i386" "arm64")
+gives="penguins-eggs"
+pkgver="9.6.21"
+homepage="https://penguins-eggs.net/"
+pkgdesc="On the road of Remastersys, Refracta, Systemback and father Knoppix!"
+maintainer="Anifyuli <anifyuliansyah@gmail.com>"
+
```
Comment (Elsie19) [inline packages/penguins-eggs-deb/penguins-eggs-deb.pacscript:8]:
```suggestion
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/penguins-eggs-deb/penguins-eggs-deb.pacscript [hunk 3]
```diff
@@ -0,0 +1,21 @@
+name="penguins-eggs-deb"
+arch=("amd64" "i386" "arm64")
+gives="penguins-eggs"
+pkgver="9.6.21"
+homepage="https://penguins-eggs.net/"
+pkgdesc="On the road of Remastersys, Refracta, Systemback and father Knoppix!"
+maintainer="Anifyuli <anifyuliansyah@gmail.com>"
+case "${CARCH}" in
```
Comment (oklopfer) [inline packages/penguins-eggs-deb/penguins-eggs-deb.pacscript:8]: change
```bash
case "${CARCH}" in
  amd64)
    url="https://sourceforge.net/projects/penguins-eggs/files/DEBS/eggs_${pkgver}_amd64.deb"
    hash="4f794a8dbe983dd710313e584dab9017b79b45ab86f06e2548e5d4bdd6c2af64"
    ;;
  arm64)
    url="https://sourceforge.net/projects/penguins-eggs/files/DEBS/eggs_${pkgver}_arm64.deb"
    hash="59fc8840bae60686e367eb96f89acddc8c42e2a773ea0b04ed601fcda1846ccb"
    ;;
  i386)
    url="https://sourceforge.net/projects/penguins-eggs/files/DEBS/eggs_${pkgver}_i386.deb"        
    hash="81c956a68a2e7e4cf9d8e74fb28149cc821a919741a732c78df1181525a8128c"
    ;;
  *) return 1 ;;
esac
```
to
```bash
case "${CARCH}" in
  amd64)
    hash="4f794a8dbe983dd710313e584dab9017b79b45ab86f06e2548e5d4bdd6c2af64"
    ;;
  arm64)
    hash="59fc8840bae60686e367eb96f89acddc8c42e2a773ea0b04ed601fcda1846ccb"
    ;;
  i386)
    hash="81c956a68a2e7e4cf9d8e74fb28149cc821a919741a732c78df1181525a8128c"
    ;;
  *) return 1 ;;
esac
url="https://sourceforge.net/projects/${gives}/files/DEBS/eggs_${pkgver}_${CARCH}.deb"    
```

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Elsie19) [inline .gitignore:5]:
```suggestion
```
@Anifyuli can you stop re-adding this line with every push?

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Comment by Elsie19** [comment]
@Anifyuli can you show a screenshot of it installing locally?

**Comment by Anifyuli** [comment]
> @Anifyuli can you show a screenshot of it installing locally?

Oh my, I deleted it from my local installation. Tomorrow I'll show you

**Comment by Elsie19** [comment]
Any updates?

**Comment by Anifyuli** [comment]
Sorry. I'm quite busy in this time, so I can't continue this pacscipt now. I'll continue in next my spare time

**Pull request closed without merging**